mindmeld/active_learning/classifiers.py

```diff
@@ -458,12 +463,22 @@ def intent_classifiers_fit_eval(
             ic.fit(queries=filtered_sampled_queries)
             # Evaluate Test Queries
             ic_eval_test = ic.evaluate(queries=filtered_test_queries)
+
             if not ic_eval_test:
-                unsampled_idx_preds_pairs.extend(
-                    (i, np.zeros(len(self.intent2idx)))
-                    for i in filtered_unsampled_queries_indices
-                )
-                continue
+                # Check for intent classifier edge cases
+                if len(domain_to_intent2id[domain]) == 1:
+                    raise ValueError(
+                        "Only one intent in domain '{!s}', use domain level tuning instead.".format(
+                            domain
+                        )
+                    )
+                else:
+                    raise ValueError(
```

Comment:
Can we just add a comment here as to why we're throwing an error when there is no test set?